Gemfile

```diff
@@ -5,7 +5,7 @@ source 'https://gems.weblinc.com' do
   gem 'workarea-ci'
 end
 
-gem 'workarea-gift_cards', github: 'workarea-commerce/workarea-gift-cards', branch: 'GIFTCARDS-7-backport-missing-via-param' # '~> 3.4.7'
+gem 'workarea-gift_cards', github: 'workarea-commerce/workarea-gift-cards', :tag => 'v3.4.7'
```

Comment:
why are you loading this from github?

Reply:
The branch previously in the file was deleted so I switched it to the version that was commented.